gauge-card/README.md

```diff
@@ -14,6 +14,7 @@ A simple gauge implemented in CSS based on https://github.com/JohnrBell/Gauge_CS
 | type | string | **Required** | `custom:gauge-card`
 | title | string | optional | Name to display on card
 | entity | string | **Required** | `sensor.my_temperature`
+| attribute | string | optional | If set, this attribute of the entity is used, instead of its state
```

Comment:
to keep this consistent with other approaches we've done, maybe just use `climate.living_room.current_temperature` as entity and check for split[2]

Reply:
Sounds legit, I will update this.

Reply:
I added dot notation as an alternative an left the explicit option in as well. Are you ok with that compromise?